### ISSUE TITLE
[Bug]:  when ever I start a song up no arrows will appear and I cant play the song/chart

### ISSUE THREAD

**Issue by sunroll1** [etternagame/etterna#1313]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Contact Details

inkchan22@gmail.com

### Version Info

Latest available release

### What operating system are you seeing the problem on?

Linux (any distro)

### Bug Behavior

the arrows for the song wont appear but I can still use my keys its just the songs wont work

### Expected Behavior

it should have worked like normal and let me play it

### Reproduction Steps

1.open the game 2.press a song ya wanna play and you will see it wont work

### Anything else?

here's the commands I used to install etterna on Linux:sudo apt update

sudo apt install kmenuedit libglu1*

wget https://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1n-0+deb10u6_amd64.deb

sudo dpkg -i libssl1.1_1.1.1n-0+deb10u6_amd64.deb

wget https://github.com/etternagame/etterna/releases/download/v0.72.3/Etterna-0.72.3-Linux.tar.gz

tar -xf Etterna-0.72.3-Linux.tar.gz

./Etterna-0.72.3-Linux/Etterna/Etterna 

kmenuedit

**Comment by poco0317** [comment]
does the music play

**Comment by sunroll1** [comment]
no the music doesnt play

**Comment by sunroll1** [comment]
yo
you gonna replie

**Comment by poco0317** [comment]
no, i dont maintain linux. the problem is linux audio related. someone else can try to figure it out

**Comment by StarbotArc** [comment]
I would simply just say try reducing the amount of apps that you have open that uses audio, as that has helped me a lot of times when I have had audio issues in Linux.

**Comment by sunroll1** [comment]
i only install etterna

**Comment by sunroll1** [comment]
will we get a flatpak version soon
